ship/src/main/scala/ch/epfl/bluebrain/nexus/ship/config/S3Config.scala

```diff
@@ -11,7 +12,7 @@ import pureconfig.generic.semiauto.deriveReader
 import java.net.URI
 import scala.annotation.nowarn
 
-final case class S3Config(endpoint: URI, importBucket: BucketName)
+final case class S3Config(endpoint: URI, importBucket: BucketName, storages: StoragesConfig, defaultBucket: BucketName)
```

Comment:
This config is technical to me (like the one about the database), the storage one which brings Delta related things would be better somewhere else